pages/doc/proxies_configuring.md

```diff
@@ -95,7 +95,7 @@ Ex: RawBlockedPoints</td>
 <td>Location of buffer files for saving failed transmissions for retry.</td>
 <td>Valid path on the local file system.<br/>
 Ex: &lt;wf_spool_path&gt;/buffer</td>
-<td>3.20</td>
+<td>The start</td>
```

Comment:
We usually leave this field blank if there is no version number. But, your choice.